pokedexForm.php

```diff
@@ -14,10 +21,12 @@
             Pokemon Type:
             <input type="string" name="addPokemonType" required="">
```

Comment:
not sure you need required="", you can just put the word required

Reply:
removed